src/LaunchDarkly.Client/Feature.cs

```diff
@@ -98,7 +98,23 @@ public bool Matches(User user)
                 var uvs = (IEnumerable<object>)userValue;
                 return Values.Intersect<object>(uvs).Any();
```

Comment:
This will be broken too, won't it? In the case that both the user and the flag rule contain multiple elements, the intersection won't do the tomfoolery defined below.